packages/design-system/src/styles/components/_Button.scss

```diff
@@ -195,14 +195,17 @@
   &.ds-c-button--disabled,
   &.ds-c-button--hover,
   &.ds-c-button--active {
-    border-color: $color-background-inverse;
+    border-color: transparent;
   }
 }
 
 .ds-c-button--inverse.ds-c-button--transparent,
 .ds-c-button--inverse.ds-c-button--transparent:visited {
   @extend %transparent-inverse-button;
 }
+.ds-c-button--inverse.ds-c-button--transparent:disabled {
+  background-color: transparent;
```

Comment:
This is the one tricky part, transparent buttons generally should be styled like links, but links dont have a disabled styling. I think disabled transparent buttons should look like other buttons, aka, it should have a background color.

Reply:
I'm not sure I follow - Are you saying you think disabled transparent buttons (links) should have a background color?

Reply:
Yes, I think the disabled transparent buttons should have a background color, I think it should look the same as the other disabled buttons.

Reasoning for this is that the transparent buttons are basically "buttons styled like links". The current disabled transparent button styling doesn't make it clear that the button is disabled, and there's no such thing as a disabled link.

Reply:
That makes sense to me. I have no problem with this given our use case. The disabled button would be that grey color, which wouldn't clash with our background.

Reply:
Is this what you'd expect? for the disabled inverse transparent button (AKA disabled link) 
![Screen Shot 2020-08-11 at 2 13 00 PM](https://user-images.githubusercontent.com/1449852/89933281-d4efe100-dbdc-11ea-8ab6-9cc493afee95.png)

Reply:
Oh, I didn't realize that it was a dark blue. I don't really care. I don't have any disabled buttons in my UtilityBelt, and if I did, I'd care more about them being clearly disabled than about the color being neutral.

Reply:
@line47 that looks almost done, I think once the underline is removed its good to go.